packages/workspace/src/browser/workspace-service.ts

```diff
@@ -66,7 +66,9 @@ export class WorkspaceService implements FrontendApplicationContribution {
     protected readonly schemaProvider: PreferenceSchemaProvider;
 
     protected applicationName: string;
-
+    protected readonly onDidCreateFileFolderEmitter = new Emitter<string>();
+    readonly onDidCreateFileFolder = this.onDidCreateFileFolderEmitter.event;
+    readonly onDidReveal = new Emitter<void>().event;
```

Comment:
- [minor] include a newline so it's easy to read.
- [minor] it'd be nice to include documentation describing the events